tests/xharness/Simulators.cs

```diff
@@ -879,8 +880,20 @@ public class Device : IDevice
 				}
 
 				// https://www.theiphonewiki.com/wiki/List_of_Apple_Watches
-				if (model.StartsWith ("Watch", StringComparison.Ordinal))
-					return Architecture.ARMv7k;
+				if (model.StartsWith ("Watch", StringComparison.Ordinal)) {
+					var identifier = model.Substring ("iPod".Length);
```

Comment:
`.Substring ("iPod")`? I think you're hitting the `default` case below when you've been testing :)

Reply:
d'oh 🙂 I actually do not have an older watch, so chances are good I'm breaking stuff...